File: Day6/Power of Two/231.cs
public class Solution 
{
    public bool IsPowerOfTwo(int n) 
    {
        int power = 1;
            
        for(int i = 0; i < 30; i++)
        {
            if(power == n)
            {
                return true;
            }

            power *= 2;
        }

        if(power == n)
        {
            return true;
        }

        return false;    
    }
}

File: Day6/Power of Two/231.py
class Solution:
    def isPowerOfTwo(self, n: int) -> bool:
        power = 1
        
        for i in range(0, 30):
            if n == power:
                return True
            
            power *= 2
            
        if n == power:
            return True
        
        return False
        